### DISCUSSION TITLE
OMDB Genre Issue

### DISCUSSION THREAD

**Discussion by undaunt** [meisnate12/Plex-Meta-Manager#589] [General]
All,

This is more of an FYI, but check out the issue I raised on the OMDB repo: https://github.com/omdbapi/OMDb-API/issues/255

The OMDB API appears to only be returning the first three genres from IMDB, so if you're like me and wanted to use the IMDB genre mass update instead of TMDB since TMDB doesn't have any (or nearly any) under Biography, Musical, etc. this is a current limitation.

Please let me know if anyone else is able to get OMDB to return more than three genres for a film that matches the IMDB website. Perhaps it's specific to types of features or some other issue.

Thanks.

**Comment by meisnate12** [comment]
thanks for letting me know

**Comment by tommargar** [comment]
I've noticed that also IMDB selected as provider doesn't return more than three genres.

/EDIT: There are only three genres max in the publicly available IMDB data.
